autobump-maven/src/main/java/com/github/autobump/maven/model/MavenVersionRepository.java

```diff
@@ -46,18 +47,31 @@ public MavenVersionRepository(String baseUrl) {
                     .collect(Collectors.toUnmodifiableSet());
         } catch (XmlPullParserException e) {
             throw new DependencyParserException("something went wrong while parseing the xml", e);
-        }catch (IOException e) {
+        } catch (IOException e) {
             return new HashSet<>();
```

Comment:
Might be ok in some cases, but be careful, you might run into an IOExceptions and when it is caught, you lose all the information about the exception by returning an empty HashSet.
In these cases the minimum I would expect is a log.warn with some information.

When returning an empty HashSet, Collections.emptySet() is your friend.